src/js/components/header/search.js

```diff
@@ -0,0 +1,38 @@
+var $ = require('jquery');
```

Comment:
I think it's OK, and that we could do the removal of jQuery in another PR since we are waiting for #1357. cc @rogeruiz @yozlet